checklist.md

```diff
@@ -93,7 +93,78 @@ User.objects.create_user(
     username=username, password=password
 )
 ```
+5. Always include `related_name` in relationships for better readability and maintainability
+
+Good example:
+```python
+class Ticket(models.Model):
+    order = models.ForeignKey(to=Order, on_delete=models.CASCADE, related_name='tickets')
+```
+
+Bad example:
+```python
+class Ticket(models.Model):
+    order = models.ForeignKey(to=Order, on_delete=models.CASCADE)
+```
+6. Use the `@transaction.atomic` decorator instead of using `with transaction.atomic:` for better readability and to ensure the entire function executes within a single transaction context
```

Comment:
```suggestion

6. Use the `@transaction.atomic` decorator instead of using `with transaction.atomic:` for better readability and to ensure the entire function executes within a single transaction context
```